ksql-engine/src/test/java/io/confluent/ksql/codegen/CodeGenRunnerTest.java

```diff
@@ -163,10 +165,11 @@ public void init() {
             new KsqlJsonTopicSerDe(), false);
         final KsqlStream ksqlStream = new KsqlStream<>(
             "sqlexpression",
-            "CODEGEN_TEST", metaStoreSchema,
-            metaStoreSchema.field("COL0"),
-            null,
-            ksqlTopic,Serdes.String());
+            "CODEGEN_TEST",
+            metaStoreSchema,
+            Optional.of(metaStoreSchema.field("COL0")),
+            new MetadataTimestampExtractionPolicy(),
+            ksqlTopic,Serdes::String);
```

Comment:
nit (not your code): two params on the same line.

Reply:
fixed.